packages/sdk/src/depositStorage/README.md

```diff
@@ -13,9 +16,9 @@ The market address defaults to market-v2-beta.mintspace2.testnet if none is prov
 
 ```typescript
 export type DepositStorageArgs = {
-    //the deposit corresponding roughly the the amounts of listings you will be doing
+    //the deposit corresponding roughly to the the amounts of listings you will be doing
```

Comment:
If we deposit with list amount set to 8 will this work exactly for 8 listings every single time? In this method, the list amount is only there to affect the deposit

Reply:
This will work for 8 listings every single time. As long as we don't (have to) tweak the amount we require per listing, that is when things might get weird. If it decreases we have a provision on the smart to allow users to claim unused storage deposit. If it increases, it becomes more expensive, but current discussions look like it's more in favor of decreasing it.

Also just spotted "the the"